shared/common-adapters/text.meta.native.tsx

```diff
@@ -3,13 +3,27 @@ import {MetaType, TextType, Background} from './text'
 
 export function defaultColor(backgroundMode: Background | null) {
```

Comment:
this file did a lot of dark/light caching which isn't aware of the ios stuff, change it all to getters